install/kubernetes/istio-rbac-beta.yaml

```diff
@@ -3,6 +3,9 @@
 # Also helps to enable logging on apiserver 'wrap' to see the URLs.
 # Each RBAC deny needs to be mapped into a rule for the role.
 # If using minikube, start with '--extra-config=apiserver.Authorization.Mode=RBAC'
+#
+# NOTE: If deploying istio to a namespace other than 'default' then change the
+# ClusterRoleBinding namspace target appropriately.
```

Comment:
We can if we scope the manager scope it's TPR list request to namespace only see initial logs in #327